storage/src/main/java/tech/pegasys/artemis/storage/Store.java

```diff
@@ -235,6 +238,16 @@ public boolean containsCheckpointState(Checkpoint checkpoint) {
     }
   }
 
+  @Override
+  public Set<Integer> getVotedValidatorIndices() {
+    readLock.lock();
+    try {
+      return votes.keySet();
```

Comment:
Does it matter that we will no longer iterate through votes in order of validator index? I don't think it will but worth being sure.

Reply:
Yep, I'm pretty sure in this scenario. The order of iterating through blocks matter but not validator indices in protoarray. Still tagging @protolambda in case he corrects me.

Reply:
Well, don't resolve the comment then, haha. Found it though. Yes, the latest set of votes is just a sum for forkchoice, and it's a set, so no duplicates. No order required :+1:

Reply:
Hahah my bad! I'm trigger happy with resolving comments :D